app/src/components/widgets/multiLevelWidgets/cumulativeTrendChart/chartjsConfig.js

```diff
@@ -23,8 +23,8 @@ const Color = require('color');
 const getDefects = (fields) => fields.filter((item) => /defects/.test(item));
 
 const getExecutions = () => [
-  'statistics$executions$passed',
   'statistics$executions$failed',
+  'statistics$executions$passed',
   'statistics$executions$skipped',
```

Comment:
I think that will be good to separate passed items from failed and skipped and not mix them (see the screenshot below), so the order will be failed, skipped, passed.

![image](https://user-images.githubusercontent.com/31861805/99373506-88e6c380-28d2-11eb-9574-af27ad23b892.png)